parsers/US_NEISO.py

```diff
@@ -248,29 +237,45 @@ def fetch_exchange(
     print("fetch_production() ->")
     pprint(fetch_production())
 
-    print('fetch_production(target_datetime=arrow.get("2017-12-31T12:00Z") ->')
-    pprint(fetch_production(target_datetime=arrow.get("2017-12-31T12:00Z")))
+    print(
+        'fetch_production(target_datetime=datetime.fromisoformat("2017-12-31T12:00:00+00:00")) ->'
+    )
+    pprint(
+        fetch_production(
+            target_datetime=datetime.fromisoformat("2017-12-31T12:00:00+00:00")
+        )
+    )
 
-    print('fetch_production(target_datetime=arrow.get("2007-03-13T12:00Z") ->')
-    pprint(fetch_production(target_datetime=arrow.get("2007-03-13T12:00Z")))
+    print(
+        'fetch_production(target_datetime=datetime.fromisoformat("2007-03-13T12:00:00+00:00")) ->'
+    )
+    pprint(
+        fetch_production(
+            target_datetime=datetime.fromisoformat("2007-03-13T12:00:00+00:00")
+        )
+    )
 
-    print('fetch_exchange("US-NEISO", "CA-QC") ->')
-    pprint(fetch_exchange("US-NEISO", "CA-QC"))
+    print(f'fetch_exchange("{US_NEISO_KEY}", "CA-QC") ->')
+    pprint(fetch_exchange(US_NEISO_KEY, "CA-QC"))
```

Comment:
```suggestion
    pprint(fetch_exchange(US_NEISO_KEY, ZoneKey("CA-QC")))
```
Just some small stuff to remove type errors from the file.